### ISSUE TITLE
Terminal doesn't responds sometimes

### ISSUE THREAD

**Issue by divyanshyadav** [mposSVK/acceptSDK#12]
This is the error log thats generated during terminal configuration check

ERROR 
ERROR - could not find protocolID for protocolString com.thyron

/BuildRoot/Library/Caches/com.apple.xbs/Sources/ExternalAccessory/ExternalAccessory-329.40.4/EASession.m:-[EASession dealloc] - 141 unable to close session for _accessory=0x1518bdbb0 and sessionID=65536

The terminal is connected.

After that we have to restart the terminal or app to start payment process again.

Is there is any way to start the terminal processes eg config check, payment without restarting the terminal or app?

**Comment by divyanshyadav** [comment]
This is the error log, for checkForTerminalConfigurationChanges function

2016-09-08 13:43:09.003 INFINIA_DEV_TARGET[457:132495] posmateExtension dealloc
2016-09-08 13:43:09.005 INFINIA_DEV_TARGET[457:132495] AcceptConfigFilesUpdateWillStart
2016-09-08 13:43:09.623 INFINIA_DEV_TARGET[457:132495] ERROR - could not find protocolID for protocolString com.thyron
2016-09-08 13:43:09.624 INFINIA_DEV_TARGET[457:132495] ERROR - /SourceCache/ExternalAccessory/ExternalAccessory-288.20.7/EASession.m:-[EASession dealloc] - 141 unable to close session for _accessory=0x15516e10 and sessionID=65536

Is this a problem of the terminal?

**Comment by WirecardMobileServices** [comment]
ERROR - could not find protocolID for protocolString com.thyron - means that you need to include com.thyron into the Supported external accessory protocols array of your application plist file - please consult section 1.4 - subsection External Accessory Protocol

**Comment by divyanshyadav** [comment]
Yes, We have added 'com.thyron' to the plist of the application but sometimes it works but sometimes doesn't and terminal also hangs up. We have tested it with two different terminals.
 Alert code is AcceptConfigFilesTerminalNotReady.
